app/views/layouts/application.html.erb

```diff
@@ -9,15 +9,16 @@
 </head>
```

Comment:
Please emphasize what's wrong here. It just works :)
BB does not care about menu now.